scripts/exglobal_archive.sh

```diff
@@ -182,12 +182,12 @@ if [[ ${HPSSARCH} = "YES" || ${LOCALARCH} = "YES" ]]; then
             targrp_list="${targrp_list} gfswave"
         fi
 
-        if [ "${DO_OCN}" = "YES" ]; then
-            targrp_list="${targrp_list} ocn_ice_grib2_0p5 ocn_ice_grib2_0p25 ocn_2D ocn_3D ocn_xsect ocn_daily gfs_flux_1p00"
+        if [[ "${DO_OCN}" == "YES" ]]; then
+            targrp_list="${targrp_list} ocean_6hravg ocean_daily ocean_grib2 gfs_flux_1p00"
```

Comment:
@aerorahul, not sure what `gfs_flux_1p00` is.

Reply:
`gfs_flux_1p00` is the coupled sflux file that @LydiaStefanova-NOAA was referring to in one of our recent GFSv17 meetings.  See [this section](https://github.com/NOAA-EMC/global-workflow/blob/842adf38087aec9f1c0bca9567e4b11d494e14c7/scripts/exglobal_atmos_products.sh#L183)

Reply:
I see. It's an atmos product. Why it's archiving with `${DO_OCN} == YES`?

Reply:
Because it was requested for evaluation when a coupled model is run (as opposed to atmosphere-only)
Having said that, I am not the expert on what is in this file or what fields are looked at by the respective teams.

Reply:
Ok, thanks!